src/rules/noUnnecessaryElseRule.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ * Copyright 2018 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as ts from "typescript";
+
+import * as Lint from "../index";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    /* tslint:disable:object-literal-sort-keys */
+    public static metadata: Lint.IRuleMetadata = {
```

Comment:
TSLint's trying to move to more descriptive documentation with code samples for rules. Please add a `codeExamples` section to this rule.

You can see rules such as `arrowReturnShorthandRule.ts` for examples of existing ones.

Reply:
Done https://github.com/palantir/tslint/pull/4502/commits/b46d5385f6d5848136581f593a1a465db4990ffe